Tests/AdyenTests/Adyen Tests/Components/Boleto/BoletoComponentTests.swift

```diff
@@ -292,19 +297,20 @@ class BoletoComponentTests: XCTestCase {
         
         sut = BoletoComponent(paymentMethod: method,
                               apiContext: Dummy.context,
+                              adyenContext: Dummy.adyenContext,
                               configuration: mockConfiguration)
         sut.delegate = mockDelegate
         
         UIApplication.shared.keyWindow?.rootViewController = sut.viewController
         
         DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + .seconds(1)) {
-        
+
             let submitButton: SubmitButton? = self.sut.viewController.view.findView(by: "payButtonItem.button") as? SubmitButton
```

Comment:
why these extra space changes are coming? Do we need them?

Reply:
nice catch, already removed them!